eslint/babel-eslint-parser/test/index.js

```diff
@@ -1,10 +1,13 @@
 import path from "path";
 import escope from "eslint-scope";
 import unpad from "dedent";
+import { fileURLToPath } from "url";
+import { createRequire } from "module";
 import { parseForESLint } from "../src";
 
 const BABEL_OPTIONS = {
-  configFile: require.resolve(
+  configFile: path.resolve(
+    path.dirname(fileURLToPath(import.meta.url)),
     "../../babel-eslint-shared-fixtures/config/babel.config.js",
   ),
```

Comment:
In many places we could use URLs directly rather than `path.*`, but only after dropping Node.js <= 10 (because we need native `fileURLToPath`):

```js
configFile: fileURLToPath(
  new URL(../../babel-eslint-shared-fixtures/config/babel.config.js, import.meta.url)
)
```